src/popup/pages/profile.tsx

```diff
@@ -8,6 +8,7 @@ import { getUserData, getUserPRData, getUserHighlightsData } from "../../utils/f
 import { emojify } from "node-emoji";
 import { goBack, goTo } from "react-chrome-extension-router";
 import { getRelativeDays } from "../../utils/dateUtils";
+import { countUniqueFullNames } from "../../utils/getContributedRepos";
```

Comment:
```suggestion
import { countUniqueRepos} from "../../utils/getContributedRepos";
```
I'd prefer this to be with Repo in the name. It is more clear. Not everyone knows about full names until you get into the API.